### PR TITLE
Define vertical structure

### PR DESCRIPTION
**Pull request by annalea-albright**
Just trying to add 'calculate_density.py' code

### CODE REVIEW

**Comment by leifdenby** [comment]
This looks great @annalea-albright!

Maybe you could add a test too (in say `tests/test_derived_profiles.py`), something like:

```python
import numpy as np


from eurec4a_environment.variables.calc_density import calc_density
import eurec4a_environment.source_data

def test_density_calculation():
    ds = eurec4a_environment.source_data.open_joanne_dataset()
    ds = ds.isel(sounding=slice(0, 10))
    da_density = calc_density.calc_peakRH_linearize(ds)
    # check that we get some sensible numbers out for the density
    assert da_density.units == 'kg/m3'
    assert np.all(da_density < 2.0)
    assert np.all(da_density > 0.0)
```

**Comment by leifdenby** [comment]
Remember to run the test by running `pytest` in the root of the repository, i.e. from the folder where `tests/` is (you will need to install `pytest` first)

```bash
$> pytest
```

**Comment by annalea-albright** [comment]
You're right, will do!

**Comment by leifdenby** [comment]
> Additionally, should we work on a quick and dirty `units` module, so that every function doesn't have to handle this independently?

I like that idea, we could also use [pint](https://pint.readthedocs.io/), but maybe that's an idea for a little down the line? I'm just thinking it might be nice to add more functionality first, but I'm also very happy for anyone to add it 😄

**Comment by Geet-George** [comment]
Yeah.. I agree. Best kept for later.

File: eurec4a_environment/variables/calc_density.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import xarray as xr
+from ...constants import Rd, Rv, eps
```
Comment (Geet-George) [inline eurec4a_environment/variables/calc_density.py:6]:
```suggestion
from ..constants import Rd, Rv, eps
```
This should solve the pytest error, I think. Not sure.

Comment (leifdenby) [reply eurec4a_environment/variables/calc_density.py:6]: Yes I think you're right :) 

```
eurec4a_environment/variables/calc_density.py:6: in <module>
    from ...constants import Rd, Rv, eps
E   ValueError: attempted relative import beyond top-level package
```

**Comment by leifdenby** [comment]
I was just about to approve this, but I'm a bit confused because you have a `calc_from_gradient` in `variables/boundary_layer/mixed_layer_height_from_gradient.py` and `variables/boundary_layer/mixed_layer_height.py`. Did you mean to add both? :)

I think you don't need the `variables/boundary_layer/mixed_layer_height_from_gradient.py` file, right?

**Comment by annalea-albright** [comment]
@leifdenby, correct, `variables/boundary_layer/mixed_layer_height_from_gradient.py` is not needed! Thanks for catching this

**Comment by leifdenby** [comment]
@annalea-albright do you want to delete the file and make another commit and push? Then I'll get this merged in :) 👍

**Pull request merged**